src/unread/unreadHelpers.js

```diff
@@ -34,7 +36,7 @@ export const removeItemsDeeply = (objArray: Object[], messageIds: number[]): Obj
   );
 };
 
-const addItemsDeeply = (input: Object[], itemsToAdd: Object[], index: number): Object[] => {
+const addItemsDeeply = (input: Object[], itemsToAdd: NumberOrObject[], index: number): Object[] => {
```

Comment:
I tried to add more concrete type for `itemsToAdd` (they are several object types and number) but it was a bit too messy for too little benefit. Using `any` instead, but in several places instead of `NumberObject`it turned out the correct type was `number`